code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -110,6 +110,21 @@
 	eat()
 	return
 
+/obj/singularity/wizard/attack_tk(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		GET_COMPONENT_FROM(insaneinthemembrane, /datum/component/mood, C)
+		if(insaneinthemembrane.sanity > 15)

```

Comment:
Shouldn't this be <15, rather than >15 if low sanity (or having it zeroed) makes them ignore it?

Reply:
shiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii